my final project step2.py

```diff
@@ -0,0 +1,134 @@
+people = {}  
+def getValidInput(input_type):
+    while True:
```

Comment:
הפונקציה הזו לא כתובה טוב, ואני רוצה שתשים לב למטרה שלה.
הפונקציה הזו צריכה בסך הכל לבדוק אם מה שהכניסו זה מספק תקין או לא.
האם זה מעניין אותה אם המספר התקין הוא גיל או ID? התשובה היא לא.
זה מעניין אותך רק לטובת ההדפסה.
תבין מה אתה רוצה שהפונקציה הזו תעשה.
מה שאני חושב שהיא צריכה לעשות, זה בסך הכל להחזיר בוליאני:
האם זה מספר תקין או לא מספר בכלל.
ואת הקסטינג לאינט לעשות בחוץ.
אתה תקבל כאן פרמטר input_type ורק תדפיס
print(input_type + " should be a number, please try again")
זה גם לא צריך להיות בלולאה